services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/resource_tracking/setup.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import FastAPI
+
+from ._models import ResourceTrackingState
+from .core import stop_heart_beat_task
```

Comment:
MINOR: `core` should be protected (i.e. `_core`) since it is used inside of this package,

Reply:
No it is actually used outside the package.
I think I had issues with exposing some methods in `__init__.py`

Reply:
ok, I've also fixed this